### DISCUSSION TITLE
New Debian Mirror Update?

### DISCUSSION THREAD

**Discussion by xclappy** [telekom-security/tpotce#1474] [General]
A new Mirror update got released on Saturday. Installing the ISO isnt working anymore.
When will there be an update to use it again? Since im not allowed to execute the makeiso.sh on my companys pc.

**Comment by t3chn0m4g3** [comment]
I just updated the [release](https://github.com/telekom-security/tpotce/releases/tag/22.04.0).

**Reply by xclappy** [reply]
thank you really much :D 
this saved my project

**Reply by cc374-ual** [reply]
Thank you!!!!! I've been struggling with this problem, I thought that I'm not doing something well...
